src/wallet/wallet.h

```diff
@@ -1185,6 +1185,15 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
 
     /** Whether a given output is spendable by this wallet */
     bool OutputEligibleForSpending(const COutput& output, const CoinEligibilityFilter& eligibility_filter) const;
+    
```

Comment:
Extra whitespace on this line is causing the linter to fail.

Reply:
Ah thank you, I'll take it as an opportunity to add comments, that should fix it